scripts/question_answering/batch/sync_batch_result.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+LOG_PATH=$1
+SAVE_DIR_NAME=${2:-squad_2.0}
+
+while read -r job_name job_id; do
+    aws s3 sync s3://gluon-nlp-dev/batch/${job_id}/temp ${SAVE_DIR_NAME}/${job_name}
+done < ${LOG_PATH}
```

Comment:
I don't think we should check in all the convenience scripts in the scripts, because this folder is user-facing.

Reply:
For me, I feel that it's fine to help reproduce the experiments for our own purpose. And also later add to the scheduled check.

Reply:
Should be resolved now.